include/caf/sensor_sampler.h

```diff
@@ -13,6 +13,26 @@ extern "C" {
 
 #include <drivers/sensor.h>
 
+enum trigger_activation_type {
```

Comment:
shorter name would do I think
`trigger_type`?

Reply:
I was thinking about best name that would not mislead the user.I wanted to highlight that this is condition for trigger to be activated. I can change this, more important for me is that later i can leave structs fields more descriptive.

Reply:
Tbh Im not the fan of that, because we would end up having:
```
 */
enum sensor_trigger_type {
	/**
```
from sensor API and

```
enum sensor_trigger {
```
from CAF sensor samper

Reply:
It can be something else but I would appreciate something shorter. This way it can be used as a prefix which is very convenient for identifying enums.

Reply:
I proposed something different, cant find something better tbh.